services/motion/client_test.go

```diff
@@ -481,21 +480,21 @@ func TestClient(t *testing.T) {
 		})
 
 		t.Run("otherwise returns a slice of PlanWithStatus", func(t *testing.T) {
-			steps := []motionplan.PlanStep{
-				{base.Named("mybase"): zeroPose},
-			}
+			steps := []motionplan.PathStep{{"mybase": zeroPoseInFrame}}
```

Comment:
The frame system keys off shortnames rather than resource names.  This is an inconsistency that we've dealt with for a while and not something I am happy about, but I don't think I can change it here

Reply:
Ok, is there a ticket to track it? I ask b/c if you have 2 components in a robot, both being used for motion planning, both with the same short name it could result in pathological behavior right?

Reply:
RDK will not allow you to have two components with the same shortname on the same robot.